bindings/netstandard/ElectionGuard/ElectionGuard.Encryption/NativeInterface.cs

```diff
@@ -150,6 +150,71 @@ internal static unsafe class NativeInterface
 
         internal unsafe struct CharPtr { };
 
+        #region Collections
+
+        internal static unsafe class LinkedList
+        {
+            internal unsafe struct LinkedListType { };
+
+            internal class LinkedListHandle
+                : ElectionguardSafeHandle<LinkedListType>
+            {
+                protected override bool Free()
+                {
+                    if (IsClosed) return true;
+
+                    var status = LinkedList.Free(TypedPtr);
+                    if (status != Status.ELECTIONGUARD_STATUS_SUCCESS)
+                    {
+                        Console.WriteLine($"LinkedList Error Free: {status}");
+                        return false;
+                    }
+                    return true;
+                }
+            }
+
+            [DllImport(DllName, EntryPoint = "eg_linked_list_new",
+                CallingConvention = CallingConvention.Cdecl, SetLastError = true)]
+            internal static extern Status New(out LinkedListHandle handle);
```

Comment:
the marshaller knows what to do with the safe handle type so we can pass that into our functions and it will automagically resolve the pointer.